### ISSUE TITLE
Please don't change BuddyPress version, leave this task to the BuddyPress core team.

### ISSUE THREAD

**Issue by imath** [rtCamp/rtMedia#2009]
Hi,

I'm one of the lead developers of BuddyPress. I'm kindly asking you to stop modifying the `buddypress()->version` as you're not the only plugin using it. Could you edit your code and package an update asap?

The `buddypress()->version` **needs to remain a string**.

What you are doing here is wrong:
https://github.com/rtCamp/rtMedia/blob/develop/app/main/controllers/template/RTMediaNav.php#L133|L134

Instead, you should do that:
```php
// to solve an issue of Media Tab is not showing in version 10.0.0.
$bp_version = floatval( $bp->version );

if ( isset( $bp_version ) && version_compare( $bp_version, '2.5.3', 'gt' ) ) {
```

**In other words please don't assign a float value to $bp->version, everyone is impacted when you do that.**

Thanks in advance to quickly take an action about it.

**Comment by pavanpatil1** [comment]
Hi, 

Thanks for reporting the issue. Our team is looking into it. 

Thanks.

**Comment by pavanpatil1** [comment]
This is fixed and released in the latest version of plugin. Hence closing this issue as done.
